server/jetstream_cluster.go

```diff
@@ -7484,6 +7553,14 @@ func (s *Server) jsClusteredConsumerRequest(ci *ClientInfo, acc *Account, subjec
 		return
 	}
 
+	node := cc.nodeForConsumerProposals(sa)
+	if node == nil {
+		// TODO(nat): is this the right thing to do?
```

Comment:
I think, yes. Probably unlikely to hit this though, the stream leader should never get to this stage since it shouldn't delete and recreate its `node`.